python/cudf/cudf/core/window/ewm.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION.
+
+from typing import Union
+
+import numpy as np
+from pandas.core.window.ewm import get_center_of_mass
```

Comment:
Is this considered part of the pandas API? @mroeschke is there a way to access this via a more public import (we've been trying to avoid pandas.core)?

Reply:
I don't think this will be apart of the pandas public API since it's only a very specific helper function. I would just vendor over this function here since it's fairly small